web/viewer.html

```diff
@@ -142,10 +142,12 @@
               <span data-l10n-id="find_next_label">Next</span>
             </button>
           </div>
-          <input type="checkbox" id="findHighlightAll" class="toolbarField">
```

Comment:
Have you tried to put the input inside the label instead of wrapping the whole checkbox set in a div? If you put the input in the label, then the `for` attribute of the label is not needed any more. Verify that the the appearance is still OK after making this change though.